### ISSUE TITLE
Where's the texi file for Python 3?

### ISSUE THREAD

**Issue by YorkZ** [Wilfred/python-info#11]

**Comment by kaushalmodi** [comment]
See https://github.com/Wilfred/python-info/pull/12

**Comment by alejandrogallo** [comment]
See maybe https://github.com/alejandrogallo/python-info

**Comment by yitang** [comment]
@alejandrogallo thanks for the update but I couldn't get your repo working. it throw error like below: 

Running Sphinx v5.3.0
building [mo]: targets for 0 po files that are out of date
building [texinfo]: all documents
updating environment: [new config] 489 added, 0 changed, 0 removed
reading sources... [  1%] c-api/arg
Exception occurred:
  File "/Users/xxx/.tmp/python-info/build/python_src/Doc/tools/extensions/pyspecific.py", line 338, in run
    label = translators['sphinx'].gettext(label)
KeyError: 'sphinx'
The full traceback has been saved in /var/folders/lj/ql496znj6y72f3nyprkly_v80000gn/T/sphinx-err-qi9tdfxc.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!

**Comment by alejandrogallo** [comment]
Yes I can confirm is not working anymore, it's probably because the sphinx version changed in the meantime,
it used to work when I did it last time. I tried to build it with sphinx `5.2.0` and it worked,
so I will change the version in the `build/Makefile` to pin it to the version, I don't have too much time to check the 5.3.0 version of sphinx

I pushed the new commit here 
https://github.com/alejandrogallo/python-info/commit/fba852bac9b5be10b746bb8caee37bc0b219d1c0
